app/views/RoomView/components/HeaderCallRightIcon.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import * as HeaderButton from '../../../containers/HeaderButton';
+import { useVideoConf } from '../../../lib/hooks/useVideoConf';
+
+export default function HeaderCallIcon({ rid }: { rid: string }): React.ReactElement | null {
```

Comment:
I see you use a separate file to use hooks, but can you call it `HeaderCallButton`?
You'll be using it on `RightButtons` component.
Just to use the same naming.